### PR TITLE
feat: RFC010: Issue and Present Organisational Credentials

### PR DESCRIPTION
**Pull request by informartin**
This specification defines the workflows for issuing and presenting Organisational Credentials.

### CODE REVIEW

**Comment by endimion** [comment]
Hey many thanks for this. Looks good. 

Although (personally) I really support DIDcomm and want a chance to play around with it more, I have some restraints into issuing an RFC that is based on it as part of EWC. Specifically, in the ARF type 2 configuration is out. My understanding on this is that only OIDC 4VC protocols can be used for implementing EUDI wallet flows. 

If I understood the pull request correctly, the apporach is based on DIDcom however... in my view this makes it incompatible with the EUDI wallet ... Or am I missing something?

**Comment by lalc** [comment]
I didn’t grasp the context behind submitting this as an RFC for EWC LSP deployments. 

I’d like to discuss it further before deciding whether or not to proceed.

**Comment by informartin** [comment]
> Hey many thanks for this. Looks good.
> 
> Although (personally) I really support DIDcomm and want a chance to play around with it more, I have some restraints into issuing an RFC that is based on it as part of EWC. Specifically, in the ARF type 2 configuration is out. My understanding on this is that only OIDC 4VC protocols can be used for implementing EUDI wallet flows.
> 
> If I understood the pull request correctly, the apporach is based on DIDcom however... in my view this makes it incompatible with the EUDI wallet ... Or am I missing something?

From our point of view, the ARF only defines the use of OIDC4VC in relation to personal wallets. For example, looking at the [Reference Architecture](https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/blob/v1.4.0/docs/arf.md#42-reference-architecture), we clearly see that it is intended to be used with a user device. However, while OID4VC is the preferred and recommended protocol for user devices, the architecture does not explicitly prohibit the use of other exchange protocols. Rather, these protocols can be defined in rulebooks for different types of attestations.

That being said, organisational wallets have very different requirements, they are expected to be available 24/7, provide endpoints, exchange credentials autonomously, support multiple users, and more. OIDC4VC is suitable for use cases involving user devices but does not address enterprise requirements. DIDComm provides persistent secure connections that enable ad hoc interaction. For example, the issuance process can take hours or even days when human interaction is required. With DIDComm, we can send a credential as soon as it is ready. Therefore, DIDComm provides more functionality that is beneficial for enterprise use cases.

In summary, since the ARF does not preclude the use of other protocols but expects that different rulebooks will be defined, we recommend using DIDComm for the issuance and presentation of organisational wallet credentials to enable a wide range of applications.

**Comment by andreasabr** [comment]
Hi, 

I did a quick review of the proposed RFC and agree with @lalc and @endimion on their comments. We have to keep in mind that we are creating these RFCs to specify what is going to be developed in the scope of EWC. Even though DIDcomm is a very interesting and, for sure, powerful technology, but it is not in line with what we have developed so far, and neither what is mentioned in the ARF. 

First, we have to work on the basics and try to implement those parts properly, and in a next step we can then think of advanced and additional technologies. And I think this next step is outside the scope of EWC. 

Thus, I would suggest having a meeting in which we can discuss how to proceed further with this RFC.

**Comment by endimion** [comment]
Agreed with @andreasabr please lets discuss over a meeting... preferably after the 26th (on vacation at the moment)

**Review by lalc — COMMENTED** [review]
Discussed with @informartin and Ronald from Spherity; also discussed with Andriana.

Proposal is to take it outside of the EWC RFC track. RFCs are strictly meant for wallet-to-wallet interactions.
